### PR TITLE
Add rem-java-21-2 course to samples

### PR DESCRIPTION
**Pull request by fabianschmauder**

### CODE REVIEW

File: src/assets/samples/courses.json [hunk 1]
```diff
@@ -313,5 +313,136 @@
...
+          "Hamcrest",
+          "Chrome Dev Tools"
+        ]
+      }, 
+      {
+        "title": "Weitere Lehrinhalte",
+        "items": [
+          "Clean Code",
+          "Arbeiten im Pairing und in Gruppen",
+          "Git-Workflow mit Github",
```
Comment (lmachens) [inline src/assets/samples/courses.json:429]:
```suggestion
          "Git-Workflow mit GitHub",
```

File: src/assets/samples/courses.json [hunk 2]
```diff
@@ -313,5 +313,136 @@
...
+        "items": [
+          "Selektoren",
+          "display: flex",
+          "display: grid"
+        ]
+      }, 
+      {
+        "title": "JavaScript",
+        "items": [
+          "ECMAScript 2020",
```
Comment (lmachens) [inline src/assets/samples/courses.json:350]: Why not 2021/next?

**Review by lmachens — CHANGES_REQUESTED** [review]
gj!
JSON needs to be prettified.

File: src/assets/samples/courses.json [hunk 3]
```diff
@@ -313,5 +313,136 @@
...
+      {
+        "title": "Testing & Debugging",
+        "items": [
+          "Test-Driven-Development",
+          "Unit-Tests mit JUnit 5",
+          "Mockito",
+          "Hamcrest",
+          "Chrome Dev Tools"
+        ]
+      },
```
Comment (christophersiem) [inline src/assets/samples/courses.json:423]: duplicative

```suggestion
```

File: src/assets/samples/courses.json [hunk 4]
```diff
@@ -313,5 +313,136 @@
...
+          "Spring Boot Web",
+          "Spring Data MongoDB",
+          "Spring Security"
+        ]
+      }, 
+      {
+        "title": "DevOps",
+        "items": [
+          "Continuous Integration (CI) mit Github Actions",
+          "Continuous Deployment (CD) mit Github Actions",
```
Comment (christophersiem) [inline src/assets/samples/courses.json:398]:
```suggestion
          "Continuous Integration (CI) mit GitHub Actions",
          "Continuous Deployment (CD) mit GitHub Actions",
```

File: src/assets/samples/courses.json [hunk 5]
```diff
@@ -313,5 +313,136 @@
...
+          "Selektoren",
+          "display: flex",
+          "display: grid"
+        ]
+      }, 
+      {
+        "title": "JavaScript",
+        "items": [
+          "ECMAScript 2020",
+          "Events DOM-Manipulation",
```
Comment (christophersiem) [inline src/assets/samples/courses.json:351]: we haven't really covered this topic

```suggestion
```

**Review by christophersiem — CHANGES_REQUESTED** [review]
🙌🏽

File: src/assets/samples/courses.json (hunk 2 above)
Comment (fabianschmauder) [reply src/assets/samples/courses.json]: changed it. Was copy paste from old certificate

**Review by lmachens — CHANGES_REQUESTED** [review]
It looks like you have too many technologies.

Use this link as example:https://nf-certifica-rem-java-2-vqgfqr.herokuapp.com/?id=721e377f-93c2-4221-8503-1c5c1bc9bdc9&course=rem-java-21-2

You can adjust it by moving some topics.

**Review by lmachens — APPROVED** [review]
Some line breaks might be confusing like in "Objektorientierte Programmierung".
But it's fine. No other issue found.

**Review by christophersiem — APPROVED** [review]
lgtm

**Pull request merged**